auth-web/src/views/auth/AcceptInviteLandingView.vue

```diff
@@ -1,13 +1,20 @@
 <template>
   <div>
-    <div v-if="!invalidInvitationToken && !tokenError && !otherError">
-      <interim-landing :summary="$t('acceptInviteLandingTitle')" :description="$t('acceptInviteLandingMessage')" icon="mdi-login-variant" showHomePageBtn="false">
-        <template v-slot:actions>
-          <v-btn v-if="!isUserSignedIn()" large link color="primary" @click="redirectToSignin()">{{ $t('loginBtnLabel') }}</v-btn>
-          <v-btn v-if="isUserSignedIn()" large link color="primary" @click="redirectToConfirm()">{{ $t('acceptButtonLabel') }}</v-btn>
-        </template>
-      </interim-landing>
-    </div>
+    <template v-if="!invalidInvitationToken && !tokenError && !otherError">
+      <div v-if="isCreateAccount">
```

Comment:
Won't this be shown for all invitees (including BCSC users)?

Reply:
Yes, it will be, and there is a condition inside the template block for the create account and bcsc users.

Reply:
Ah okay so `isCreateAccount` is what hides it.  Can we rename that to something more specific to Director's Search? Technically, BCSC users create accounts as well, which is why I got confused.

Reply:
I agree, will change the variable name